src/codeu/chat/client/commandline/Chat.java

```diff
@@ -124,10 +131,16 @@ public void invoke(List<String> args) {
       }
     });
     
+    //panel.register("info", new Panel.Command() {
```

Comment:
Uncomment if necessary, delete if not. Can you make sure both features relying on variations of "ServerInfo" still work?